generators/ruby-v2/base/src/context/AbstractRubyGeneratorContext.ts

```diff
@@ -12,6 +12,21 @@ import { RubyTypeMapper } from "./RubyTypeMapper";
 
 const defaultVersion: string = "0.0.0";
 
+/**
+ * Converts a snake_case name to normalcase by removing underscores before numbers.
+ * This is required to satisfy Rubocop's Naming/VariableNumber rule which expects
+ * normalcase for symbol numbers (e.g., `first6` instead of `first_6`).
+ *
```

Comment:
Can we make this less intrusive? Supposedly the regex that rubocop is using is just this:

```
      NORMAL_CASE = /^@{0,2}-{0,1}_{0,1}[a-zA-Z\d]*(_[a-zA-Z]+\d*)*
                     [_\D]*[!?=]?$/x
```